src/components/form-control-icons/form-control-icons.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../icon/icon.jsx';
+import cautionSvg from '../../svgs/caution.svg';
+import styles from './form-control-icons.css';
+
+export default function FormControlIcons({
+  children,
+  className,
+  validationMessage,
+}) {
+  return (
+    <div className={className}>
+      {!!validationMessage && (
+        <Icon
```

Comment:
Now that's this is in a separate file, something is itching in my head that there should be some HTML or ARIA relationship between the icon and the control. But I don't know what that is. /shrug

Reply:
I've done some digging, and I agree, but no attributes jump out to me as obvious candidates. 😬